pkgs/tools/X11/xpra/default.nix

```diff
@@ -119,9 +124,9 @@ in buildPythonApplication rec {
   };
 
   meta = {
-    homepage = "http://xpra.org/";
+    homepage = "https://xpra.org/";
     downloadPage = "https://xpra.org/src/";
-    downloadURLRegexp = "xpra-.*[.]tar[.]xz$";
+    downloadURLRegexp = "xpra-.*[.]tar[.][gx]z$";
```

Comment:
Is that info actually used?

Reply:
I think it must be used by the updater script? (pkgs/tools/X11/xpra/update.sh). I added gz, because I noticed some releases were .tar.gz, and some were .tar.xz